src/dlgTriggerEditor.cpp

```diff
@@ -5498,8 +5501,22 @@ void dlgTriggerEditor::saveKey()
 
         if (pT->state()) {
             clearEditorNotification();
-            pItem->setIcon(0, icon);
-            pItem->setText(0, name);
+            if (old_name == tr("New key")) {
+                QIcon _icon;
```

Comment:
:nauseated_face:
Don't use leading `_` on identifiers - that is straying into dodgy territory as:
> * All identifiers that begin with an underscore are always reserved for use as identifiers with file scope in both the ordinary and tag name spaces.

Furthermore:
> * All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

See, e.g. https://stackoverflow.com/q/228783/4805858

In this context you should just assign the wanted icon to the original `icon` - as the scope of such an assignment is contained to the relevant `if ()` segment of code (i.e. new lines 5505 to 5515).

Reply:
This is following an existing code pattern that is applied to triggers, aliases, etc. Lets leave the style changes to style PRs 🙂

Reply:
:slightly_frowning_face: I'm sure I specially crafted an issue about the use of leading `_` on identifiers - but it looks like I never got it raised. Guess this will just have to be added to the things to fix for it...